src/main/java/seedu/address/logic/Messages.java

```diff
@@ -30,6 +30,18 @@ public class Messages {
     public static final String MESSAGE_INVALID_DATE_FORMAT = "Dates should be in yyyy-MM-dd format";
     public static final String MESSAGE_INVALID_TIME_FORMAT = "Times should be in HH:mm format";
 
+    public static final String MESSAGE_NO_SCHEMES_AVAILABLE = "There are no schemes available for this family";
```

Comment:
I think the naming convention should still reference "person" rather than family even though that person is the representative for the family as a lot of the other commands still reference the entrants as individual persons.